core/getting-started/unit-testing-using-nunit/PrimeService.Tests/PrimeService_IsPrimeShould.cs

```diff
@@ -6,9 +7,10 @@ namespace Prime.UnitTests.Services
     [TestFixture]
     public class PrimeService_IsPrimeShould
     {
-        private readonly PrimeService _primeService;
+        private PrimeService _primeService;
```

Comment:
Why are you removing the `readonly` modifier here?

Reply:
Because `_primeService` gets instantiated in the `SetUp` method.